locales/English/intro.php

```diff
@@ -113,7 +113,7 @@
 
 	<li class="formlisttitle">Move</li>
 	<li class="formlistdesc">
-		The unit tries to move into(/attack) an adjacent territory.
+		The unit tries to move into (or attack) an adjacent territory.

```

Comment:
into (also know as attacking) an adjacent
might be better since they're the same thing, not different.

Reply:
"(or if it's occupied, attack)"
Seems to match better?